db/schema.rb

```diff
@@ -68,9 +68,9 @@
     t.integer "income_other"
     t.boolean "has_home"
     t.boolean "has_vehicle"
-    t.text "financial_accounts", default: [], array: true
     t.boolean "sms_reminders"
     t.boolean "email_reminders"
+    t.text "financial_accounts", default: [], array: true
```

Comment:
I'm not sure what this is, I'm happy to remove this from the PR - just let me know.